package.nls.json

```diff
@@ -19,6 +19,7 @@
     "DataScience.insecureSessionDenied": "Denied connection to insecure server.",
     "DataScience.installPython": "Python is not installed. \nPlease [download](https://www.python.org/downloads) and install Python in order to execute cells in this notebook.",
     "DataScience.installKernel": "The Jupyter Kernel '{0}' could not be found and needs to be installed in order to execute cells in this notebook.",
+    "DataScience.newNotebookUI": "Welcome to VS Code's new notebook experience!  We think you'll find it faster and more pleasing to use! To learn more, click [here](https://aka.ms/NewNotebookUI)",
```

Comment:
My minor vote is this is two exclamation points too many. But I'll defer to @greazer and @claudiaregio on the text. Code look solid to me.